custom/icds/location_reassignment/tests/test_processor.py

```diff
@@ -37,25 +36,18 @@ def test_process(self, location_types_mock, locations_mock, deprecate_locations_
             },
             'state': {}
         }
-        Processor(self.domain, transitions, site_codes).process()
-        calls = [call([location_112], [location_131], MOVE_OPERATION),
-                 call([location_12], [location_13], MOVE_OPERATION)]
+
+    def test_process(self, locations_mock, deprecate_locations_mock, *_):
```

Comment:
`*args` seems more standard, although I see other places we use `*_`.

Reply:
that other places might be me only. I tend to use `*_` where i don't care about whats passed in the args at all.

Reply:
It reminds me of perl, in not the best way :-p

Reply:
no knowledge of it :)